assets/css/3-layouts/_video.sass

```diff
@@ -94,7 +94,7 @@
             border-radius: 0 0 3px 3px
             overflow: hidden
             transition: all 100ms
-            background: rgba(145, 141, 141, 0.65)
+            background: #a557f147
```

Comment:
This is not a valid Hex value.
Please also check color contrast to make sure it's above 4.5:1 radio for white text.

Reply:
i used [this](https://contrast-ratio.com/#white-on-rgba%28162%2C%2083%2C%20200%2C%200.37%29) site to check the ratio and got it to a ratio the site said was good.